cockatrice/src/carddbparser/cockatricexml3.cpp

```diff
@@ -23,7 +23,7 @@ bool CockatriceXml3Parser::getCanParseFile(const QString &fileName, QIODevice &d
     QXmlStreamReader xml(&device);
     while (!xml.atEnd()) {
         if (xml.readNext() == QXmlStreamReader::StartElement) {
-            if (xml.name() == COCKATRICE_XML3_TAGNAME) {
+            if (xml.name().toString() == COCKATRICE_XML3_TAGNAME) {
```

Comment:
we call xml.name().toString() so often it's obnoxious, we should just split this to a variable